src/testRunner/unittests/convertToAsyncFunction.ts

```diff
@@ -1194,6 +1194,20 @@ const [#|foo|] = function () {
 } 
 `);
 
+    _testConvertToAsyncFunction("convertToAsyncFunction_catchBlockUniqueParams", `
+function [#|f|]() {
+	return Promise.resolve().then(x => 1).catch(x => "a").then(x => !!x); 
+} 
+`);
+
+    _testConvertToAsyncFunction("convertToAsyncFunction_bindingPattern", `
+function [#|f|]():Promise<void> {

```

Comment:
>:Promise<void> [](start = 17, length = 14)

Is the return type important?  It appears to be preventing the creation of a corresponding JS test.

Reply:
Removed.